src/shared/common.js

```diff
@@ -118,6 +118,10 @@ $common.formatAnalyzerLabel = function formatAnalyzerLabel(analyzer, vulnSource,
       analyzerLabel = "VulnDB";
       analyzerUrl = "https://vulndb.cyberriskanalytics.com/vulnerabilities/" + vulnId;
       break;
+    case 'SNYK_ANALYZER':
+      analyzerLabel = "Snyk";
+      analyzerUrl = "https://security.snyk.io/vuln/" + vulnId;
```

Comment:
This needs to be added in the `case 'INTERNAL_ANALYZER'` branch a few lines above as well, when `vulnSource == 'SNYK'`. 

Because the API server is mirroring vulnerable versions, snyk vulns may be identified by the internal analyzer, too. In that case, there's no reference URL to Snyk rendered in the UI right now:
![image](https://user-images.githubusercontent.com/5693141/196929501-34e509eb-c2d0-4ca0-a540-e8757bb07653.png)

Versus when the vulnerabilities are first identified by the Snyk analyzer:
![image](https://user-images.githubusercontent.com/5693141/196929659-bf978c59-0fec-47c0-84bf-fa53060e3f77.png)